scripts/scripts.js

```diff
@@ -126,13 +126,19 @@ function OrdinalSuffix(i) {
 }
 
 // build a row of history
-function HTMLForRow(stepNumber, math, annotation, showTrash) {
+function HTMLForRow(stepNumber, math, annotation, showTrash, showStepLabel) {
 	let html = '<div class="row mathStep" data-step="'+stepNumber+'" data-equation="'+math+'" data-annotation="'+annotation+'">';
-	html += '<div class="col-md-6">';
+	html += '<div class="col-md-1">';
 	html +=   '<span role="heading" aria-level="3">';
-	html +=     '<span class="SROnly">' + OrdinalSuffix(stepNumber) +' step</span>';
-	html +=     '<span class="stepHeader" aria-hidden="true">Step '+stepNumber+':</span>';
+	if (showStepLabel) {
+		html +=     '<span class="SROnly">' + OrdinalSuffix(stepNumber) +' step</span>';

```

Comment:
I believe white spaces after '+=' were used to display HTML tags in code with an indentation, but as you used 'if' statement, it's no longer needed.

Reply:
Good catch!